packages/client/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "automattic/jetpack-client",
+	"description": "Jetpack Client",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-options": "@dev"
+	},
+	"require-dev": {
+		"phpunit/phpunit": "^5.7 || ^6.5 || ^7.5",
+		"php-mock/php-mock": "^2.1"
```

Comment:
Seems like this one isn't used - are we planning to add some tests where it will be used?